src/metabase/driver/sql/query_processor.clj

```diff
@@ -656,53 +656,53 @@
 
 (defmethod ->honeysql [:sql :between]
   [driver [_ field min-val max-val]]
-  [:between (->honeysql driver field) (->honeysql driver min-val) (->honeysql driver max-val)])
+  (hsql/call :between (->honeysql driver field) (->honeysql driver min-val) (->honeysql driver max-val)))
```

Comment:
any reasons we need to change all of this to `hsql/call`?

Reply:
I was trying to get booleans working in custom expressions but it didn't work. removed